app/assets/stylesheets/common/_icons.scss

```diff
@@ -59,3 +61,7 @@ a[role='button'] > .fa-layers {
   height: 20px;
   margin-right: 1em;
 }
+
+.fa-xmark {
```

Comment:
Don't change global icon colours without checking with me first (in this case you can share a screenshot of what you're trying to achieve)